plugins/llvm/llvm_disasm.cpp

```diff
@@ -223,13 +325,18 @@ class llvm_disassembler : public disassembler_interface {
     }
 
     void set_memory(memory m) {
+        #if LLVM_VERSION_MAJOR == 3 && LLVM_VERSION_MINOR >= 5
         mem = memory(m);
+        #else
+        mem.reset(new MemoryObject(m));
```

Comment:
It would be nice, if we can get rid of two different memory representations in two version, by depending on common abstraction.